pages/pipelines/command_step.md.erb

```diff
@@ -2,26 +2,113 @@
 
 <%= toc %>
 
-A *command* step runs a command on an agent. The following is the most simple *command* step:
+
+A *command* step runs a command on an agent. The command can be the name of a script, or <cli things>
```

Comment:
I think people are probably fine with using && in there, but it would be nice to support the list as well. And yeah, allowing plural commands would be great too!